auth_api_key_server_env/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptcamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Auth API key server environment",
+    "summary": """
+Configure api keys via server env.
+
+This can be very useful to avoid mixing your keys between your various
+environments when restoring databases. All you have to do is to add a new
+section to your configuration file according to the following convention:
+    """,
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/server-auth",
+    "author": "Camptcamp,Odoo Community Association (OCA)",
+    "depends": ["auth_api_key", "server_environment"],
```

Comment:
Adding `auto_install : True` would be nice

Reply:
No auto_install.... IMO it's a choice from the dev/ana to store the api keys into server env even if others informations are managed in this way.

Reply:
yep, that's the reason ;)